py-rse/bash-basics.Rmd

```diff
@@ -866,100 +862,107 @@ This options works the same way with `mv` and `cp`.
 
 ## How can I run commands on lots of files at once? {#py-rse-bash-basicswildcard}
 
-The `cleaned` directory in our climate data project contains
-precipitation and temperature records from four weather stations in Australia:
+The `data` directory in our zipfs-law project contains
+the text files for several ebooks from Project Gutenberg:
```

Comment:
This can be [Project Gutenberg][project-gutenberg] to link to the glossary entry